src/_posts/platform/internals/2000-01-01-regions.md

```diff
@@ -1,48 +1,48 @@
 ---
 title: Regions
-modified_at: 2020-08-03 00:00:00
+modified_at: 2022-04-26 00:00:00
 tags: internals regions
 index: 10
 ---
 
 Scalingo is available on multiple regions. Here is the list of publicly
 available regions:
 
-| Name  | Provider | API Endpoint | Dashboard | Database API Endpoint |
-| ------------- | ------------- | ------------- | ------------- | -------- |
-| osc-fr1  | [3DS Outscale](https://outscale.com/) | [link](https://api.osc-fr1.scalingo.com) | [link](https://my.osc-fr1.scalingo.com) | [link](https://db-api.osc-fr1.scalingo.com) |
-| osc-secnum-fr1  | [3DS Outscale](https://outscale.com/) | [link](https://api.osc-secnum-fr1.scalingo.com) | [link](https://my.osc-secnum-fr1.scalingo.com) | [link](https://db-api.osc-secnum-fr1.scalingo.com) |
+| Name           | Provider                              | Provider Region     | Location      |
+| -------------- | ------------------------------------- | ------------------- | ------------- |
+| osc-fr1        | [3DS Outscale](https://outscale.com/) | eu-west-2           | Paris, France |
+| osc-secnum-fr1 | [3DS Outscale](https://outscale.com/) | cloudgouv-eu-west-1 | Paris, France |
+
 {: .table }
 
 
 It is possible to [migrate applications and data between regions]({% post_url platform/app/2000-01-01-regions-migration %}).
 
-The `osc-secnum-fr1` region is only available upon request on the support.
+{% note %}
+The `osc-secnum-fr1` region is restricted and is only available upon request
+on the support.
+
+More informations on this [page]({% post_url security/procedures/2000-01-01-secnumcloud %}).
+{% endnote %}
 
 ## Accessing a Specific Region
 
 ### Using the Web Dashboard
 
-When using the web dashboard at
-[https://my.scalingo.com](https://my.scalingo.com), you will automatically be
-redirected to the region you have access to. If your account has access to
-multiple regions, a selection screen will be presented to let you chose which
-region you want to use:
-
-{% assign img_url = "https://cdn.scalingo.com/documentation/internals/screenshot_dashboard_regions_selection.png" %}
```

Comment:
Can you also remove this screenshot from the CDN as it's not used anywhere else?

Reply:
It will be deleted later in the day